docs/crd_mysql_backup_schedule.md

```diff
@@ -0,0 +1,44 @@
+# BackupSchedule
+
+`BackupSchedule` is a custom resource definition (CRD) that represents
+a full dump & binlog scheduling.
+
+`BackupSchedule` creates `CronJob` which collects binlog and then rotate dump.
+
+| Field        | Type                                          | Description                                          |
+| ------------ | --------------------------------------------- | ---------------------------------------------------- |
+| `apiVersion` | string                                        | APIVersion.                                          |
+| `kind`       | string                                        | Kind.                                                |
+| `metadata`   | [ObjectMeta]                                  | Standard object's metadata.                          |
+| `spec`       | [BackupScheduleSpec](#BackupScheduleSpec)     | Specification of scheduling.                         |
+| `status`     | [BackupScheduleStatus](#BackupScheduleStatus) | Most recently observed status of the scheduled jobs. |
+
+## BackupScheduleSpec
+
+| Field                    | Type                                    | Required | Description                                                               |
+| ------------------------ | --------------------------------------- | -------- | ------------------------------------------------------------------------- |
+| `clusterName`            | string                                  | Yes      | Name of [`MySQLCluster`](crd_mysql_cluster.md)                            |
+| `schedule`               | string                                  | Yes      | Schedule in Cron format, this value is passed to `CronJob.spec.schedule`. |
+| `objectStorageEndpoint`  | [ObjectStorageSpec](#ObjectStorageSpec) | Yes      | Specification of S3 compatible object storage.                            |
+| `retentionPeriodSeconds` | int                                     | Yes      | Retention period of each backup file.                                     |
```

Comment:
Specifying duration in seconds is a bit troublesome.
For instance, how many seconds a week is?

I'd like to recommend to use [metav1.Duration](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1?tab=doc#Duration) instead.

Reply:
Changed to Duration.